### PR TITLE
use `randomUUID()` instead of `Math.random()` for generating workflow ids

### PR DESCRIPTION
**Pull request by vkarpov15**
Re: temporalio/samples-typescript#110

## What does this PR do?

A couple of examples in this blog post use `Math.random()` to generate workflow ids. This is bad practice, we don't want people using `Math.random()` to generate workflow ids in production. This PR uses [Node's built-in `randomUUID()` function](https://nodejs.org/api/crypto.html#cryptorandomuuidoptions) instead to provide unique workflow ids without additional dependencies.

### CODE REVIEW

File: blog/2022-02-28-temporal-rest.md [hunk 1]
```diff
@@ -48,13 +48,14 @@ To execute an instance of `counterWorkflow`, use `WorkflowClient` to start the W
...
 import { counterWorkflow } from './workflows';
+import crypto from 'crypto';
 
 async function run() {
   const client = new WorkflowClient();
 
   const handle = await client.start(counterWorkflow, {
     taskQueue: 'tutorial',
-    workflowId: 'wf-id-' + Math.floor(Math.random() * 1000),
+    workflowId: 'workflow-' + crypto.randomUUID(),
```
Comment (mfateev) [inline blog/2022-02-28-temporal-rest.md:58]: I think the workflow id should come from some business id. What is the valid business id in this case?

Comment (lorensr) [reply blog/2022-02-28-temporal-rest.md:58]: There isn't an existing business ID. This use case is just "start a run of the counter workflow from a command-line script for example purposes". Perhaps:

```suggestion
    // Usually, we use a business ID. In this example, we don't have one, so we generate an ID.
    workflowId: 'counter-' + crypto.randomUUID(),
```

**Review by lorensr — APPROVED** [review]
Example IDs can be updated to `counter-*`

image

image

**Review by vkarpov15 — COMMENTED** [review]
@lorensr thanks for your suggestions :+1:

**Review by djmagee — APPROVED** [review]

**Pull request merged**